### ISSUE TITLE
Unable to edit Username Field

### ISSUE THREAD

**Issue by ColinTheBossReal** [getgrav/grav-plugin-admin#2344]
So this is the current problem I am having after just installing Grav. When I go to create a User it won't let me edit the username field

![image](https://user-images.githubusercontent.com/57803845/220479047-41717f3a-aaef-489e-81c2-ff857f60f423.png)

And as you can see in the image it is set to be disabled and read only after hitting add account. If I edit the HTML here it will allow me to create a new user just fine.

**Comment by rhukster** [comment]
Yes that is a limitation as the userid is infact the "id" of the user you can't change that.  just create a new user with the username you want OR you can jsut rename the file manually in the `/user/accounts/` folder.

**Comment by ColinTheBossReal** [comment]
But it will not let you create a user without a username

**Comment by ColinTheBossReal** [comment]
![image](https://user-images.githubusercontent.com/57803845/220479893-34134a35-487c-48bf-97df-445621d6d75b.png)

**Comment by rhukster** [comment]
Actually apologies, this is a bug. Fixed now and releasing 1.7.39.4 with fix.
